libtenzir/test/pipeline.cpp

```diff
@@ -192,6 +194,10 @@ struct fixture : fixtures::deterministic_actor_system_and_events {
         if (not diag_error and d.severity == severity::error) {
           diag_error = caf::make_error(ec::unspecified, fmt::to_string(d));
         }
+      },
+      [&](pipeline_op_metrics&) {
+        MESSAGE("received metrics");
+        // Do nothing with metrics locally for now.
```

Comment:
At least some basic assertions should be made here. Can we expect that we get data from all operators in the pipeline?

Reply:
>Can we expect that we get data from all operators in the pipeline?

Yes, up until the executor has finished.
I added a check in https://github.com/tenzir/tenzir/pull/3376/commits/26b4bed73aac34962561ba60dd034dad12e10be3